.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,123 @@
+      description: When did the issue last occur?
+      placeholder: "Example: 'Jul 10, 2024 12:08:00 AM UTC'"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps_to_reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: How can we reproduce the bug?
+      placeholder: |
+        Example: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)